lighthouse-cli/cli-flags.js

```diff
@@ -137,9 +137,17 @@ function getFlags(manualArgv) {
           default: 'localhost',
           describe: 'The hostname to use for the debugging protocol.',
         },
-        'emulated-form-factor': {
+        'form-factor': {
           type: 'string',
-          describe: 'Controls the emulated device form factor (mobile vs. desktop) if not disabled',
+          describe: 'Determines how performance metrics are scored and if mobile-only audits are skipped. For desktop, --preset=desktop instead.',
+        },
+        'screenEmulation': {
+          describe: 'Sets screen emulation parameters. See also --preset. Use --screenEmulation.disabled to disable. Otherwise set these 4 parameters individually: --screenEmulation.mobile --screenEmulation.width=360 --screenEmulation.height=640 --screenEmulation.deviceScaleFactor=2',
```

Comment:
purposefully don't want to break these up like `throttling`?

Reply:
correct. I don't wanna grow --help even more.